FoxHound.Web/appsettings.Test.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I didn't know is this environment for Azure should be considered Test...but that's what i went with

Reply:
Makes sense.  My guess is the plan is to configure the instrumentation key via the configuration in the app service.  Consider an empty placeholder in the root appsettings for this just so its obvious what configuration exists.

Reply:
Yes, i will configure the key in configuration in the app service. Good idea about adding placeholder...it's been added.